docs/development/extensions-core/postgresql.md

```diff
@@ -27,6 +27,8 @@ To use this Apache Druid extension, [include](../../configuration/extensions.md#
 
 ## Setting up PostgreSQL
 
+To avoid issues with upgrades that require schema changes to a large metadata table, chose a PostgreSQL version that supports instant ADD COLUMN semantics.
```

Comment:
```suggestion
To avoid issues with upgrades that require schema changes to a large metadata table, choose a PostgreSQL version that supports instant ADD COLUMN semantics.
```